learning_observer/learning_observer/auth/events.py

```diff
@@ -321,6 +329,11 @@ async def authenticate(request, headers, first_event, source):
     for auth_method in learning_observer.settings.settings['event_auth']:
         auth_metadata = await AUTH_METHODS[auth_method](request, headers, first_event, source)
         if auth_metadata:
+            auth_response = auth_metadata.get('auth_response')
+            if auth_response and "status_code" in auth_response and auth_response.get("status_code") == 403:
+                print("Forbidden.")
```

Comment:
This ought to be included in the logger (`debug_log()`) with more information about why. We could run into problems if the blacklisting/whitelisting pattern is broken. Including this in our debug logs could help quicker pinpoint the issue.

Reply:
I've replaced this print statement with `debug_log()` instead